.idea/vcs.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
while I really appreciate your decision not to use VS Code, you shouldn't commit any files related to the IDE. Please add this folder to .gitignore

Reply:
Same goes for "chromedriver" binary, it shouldn't be in a repository

Reply:
Thanks for review. I have deleted those stuff from repo and added them to gitignore.

Look more at commits:

- [add .idea folder to gitignore ](https://github.com/d9nchik/selenium/pull/1/commits/e2acf2a15aaef68146641242bf0139b88038a858)
- [remove chromedriver from repo](https://github.com/d9nchik/selenium/pull/1/commits/edc6898e32462d4273bed4bf36f781129c96ff57)